core/src/main/java/org/fao/geonet/kernel/datamanager/base/BaseMetadataSchemaUtils.java

```diff
@@ -0,0 +1,122 @@
+package org.fao.geonet.kernel.datamanager.base;
+
+import java.nio.file.Path;
+import java.util.Set;
+
+import javax.annotation.CheckForNull;
+
+import org.fao.geonet.constants.Geonet;
+import org.fao.geonet.domain.Metadata;
+import org.fao.geonet.exceptions.NoSchemaMatchesException;
+import org.fao.geonet.exceptions.SchemaMatchConflictException;
+import org.fao.geonet.kernel.SchemaManager;
+import org.fao.geonet.kernel.datamanager.IMetadataSchemaUtils;
+import org.fao.geonet.kernel.schema.MetadataSchema;
+import org.fao.geonet.repository.MetadataRepository;
+import org.fao.geonet.utils.Log;
+import org.jdom.Element;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import jeeves.server.context.ServiceContext;
+
+public class BaseMetadataSchemaUtils implements IMetadataSchemaUtils {
+
+    @Autowired
+    private SchemaManager schemaManager;
+
+    @Autowired
+    private MetadataRepository metadataRepository;
+
+    public void init(ServiceContext context, Boolean force) throws Exception {
+//        schemaManager = context.getBean(SchemaManager.class);
```

Comment:
I think `@Autowired` will not work due to multinode. Please check to use again `context.getBean`

Reply:
Fixed